kubectx

```diff
@@ -94,19 +97,31 @@ save_context() {
   saved="$(read_context)"
 
   if [[ "${saved}" != "${1}" ]]; then
-    printf %s "${1}" > "${KUBECTX}"
+    printf %s "${1}" > "${KUBECTX_PREVIOUS_VALUE}"
   fi
 }
 
 switch_context() {
   $KUBECTL config use-context "${1}"
 }
 
+context_list_count() {
+  if [[ -f ${CONTEXT_LIST_CACHE} ]]
+  then
+    echo $(($(cat ${CONTEXT_LIST_CACHE} | wc -l) + 2))
+  else
+    echo "100%"
+  fi
+}
+
 choose_context_interactive() {
+  list_contexts > /dev/null
+  HEIGHT="$(context_list_count)"
+
   local choice
   choice="$(_KUBECTX_FORCE_COLOR=1 \
```

Comment:
sadly by changing `SELF_CMD=` to `cat $f`, you're losing the functionality that highlights the current context in fzf screen. :(

we pass env var `_KUBECTX_FORCE_COLOR=1` to fzf (as it used to call `kubectx`, which uses that env var to highlight the current context even when the output is not an interactive tty).

I don't know how we can keep that functionality and read from the cache.
At this point, I'm wondering if `fzf` could do `--height=auto` somehow. Do you mind opening an issue to `fzf` to discuss that possibility?

Reply:
I think we can find a solution. That would be a shame to rely on some feature implemented and accepted in `fzf`: it can never happen.

Reply:
But have we tried? Maybe it would be accepted and implemented pretty quickly.